pyproject.toml

```diff
@@ -16,6 +16,7 @@ pyarrow = ">3.0.0"
 sqlite-shelve = "^2.0.1"
 typing-extensions = "^4.8.0"
 h5py = "^3.10.0"
+google-cloud-storage = "1.35.0"
```

Comment:
To make this play nicer with other libraries, I'd recommend specifying this as "^1.35.0" instead. (The lockfile will pin it to a specific version so you don't have to.)